mlflow/models/evaluation/default_evaluator.py

```diff
@@ -258,6 +261,59 @@ def _do_plot(**kwargs):
 }
 
 
+def _evaluate_custom_metric(index, custom_metric, eval_df, builtin_metrics):
+    """
+    This function calls the `custom_metric` function and performs validations on the returned
+    result to ensure that they are in the expected format. It will raise a MlflowException if
+    the result is not in the expected format.
+
+    :param index
+    :param custom_metric: A user provided custom metric function
+    :param eval_df: A Pandas dataframe object containing a prediction and a target column
+    :param builtin_metrics: A dictionary of metrics produced by the default evaluator
+    :return: A tuple of dictionaries. The first is a dictionary of metrics, the second is
+             a dictionary of artifacts (which can be null if the custom metric function did
+             not produce any).
+    """
+    result = custom_metric(eval_df, builtin_metrics)
+    if result is None:
```

Comment:
We raise on incorrect return format from `custom_metric` function. Why is None considered special here since it's not a correct format?

Reply:
Oh, because I was following this https://docs.google.com/document/d/1lC2PMENiT4G7o-QklRdeukGBON_hv9D47G1Q5GCt3GE/edit#bookmark=id.cod7z43dnwqy. I realized I didn't state this in the design document, should I just remove this behavior?